src/Classifiers/ControllerClassifier.php

```diff
@@ -16,7 +17,11 @@ public function satisfies(ReflectionClass $class)
     {
         return collect(resolve(Router::class)->getRoutes())
             ->map(function ($route) {
-                return get_class($route->getController());
+                try {
+                    return get_class($route->getController());
+                } catch (Exception $e) {
+                    return;
+                }
```

Comment:
use this instead of `try catch`

```php
return collect(resolve(Router::class)->getRoutes())
    ->filter(function ($route) {
        return is_string($route->action['uses']);
    })
    ->map(function ($route) {
        return get_class($route->getController());
    })
    ->unique()
    ->contains($class->getName());
```